pkg/cloudprovider/aws/apis/v1alpha1/register.go

```diff
@@ -47,6 +49,15 @@ var (
 	ResourceAMDGPU    v1.ResourceName = "amd.com/gpu"
 	ResourceAWSNeuron v1.ResourceName = "aws.amazon.com/neuron"
 	ResourceAWSPodENI v1.ResourceName = "vpc.amazonaws.com/pod-eni"
+
+	InstanceFamilyLabelKey          = LabelDomain + "/instance.family"
+	InstanceSizeLabelKey            = LabelDomain + "/instance.size"
+	InstanceCPULabelKey             = LabelDomain + "/instance.cpu"
+	InstanceMemoryLabelKey          = LabelDomain + "/instance.memory"
+	InstanceGPUNameLabelKey         = LabelDomain + "/instance.gpu.name"
+	InstanceGPUManufacturerLabelKey = LabelDomain + "/instance.gpu.manufacturer"
+	InstanceGPUCountLabelKey        = LabelDomain + "/instance.gpu.count"
+	InstanceGPUMemoryLabelKey       = LabelDomain + "/instance.gpu.memory"
```

Comment:
should add `hypervisor` here as well. It's common to select only `nitro` instance types and it's not straightforward to map instance families to `nitro` since gen numbers aren't uniform.

Reply:
Can we address in a followon?